enterprise/api_client/discovery.py

```diff
@@ -121,6 +121,29 @@ def get_search_results(self, querystring=None, traverse_pagination=True):
             many=False,
         )
 
+    def get_catalog_results(self, query):
+        """
+        Return results from the discovery service's search/all endpoint.
+
+        Arguments:
+            query (dict): query parameters used to filter search results.
+
+        Returns:
+            dict: The paginated response.
+        """
+        response = {}
+
+        try:
+            endpoint = getattr(self.client, self.SEARCH_ALL_ENDPOINT)
+            response = endpoint().post(query)
```

Comment:
I know I'm jumping in a little randomly here, but I discovered a bug on the business sandbox where our enterprise catalog api is not properly handling pagination - it always returns the first page of results no matter what page you pass in. After some digging, I ended up here. We need to be able to pass pagination params to the discovery service in order to support pagination for the enterprise api - I believe the function above (`get_search_results`) does this, so that needs to be replicated here somehow. Let me know if you have any questions!

Reply:
@brittneyexline Thanks a lot for reporting the bug and sorry for any inconvenience caused by this. I will fix the bug.